### ISSUE TITLE
How to configure DNS records?

### ISSUE THREAD

**Issue by Intel1146** [pboehm/ddns#4]
Hello.

Is the domain supposed to use the server as its NS, or just point d.domain.com to the IP? I'm a bit confused because it seems as if I should point NS to the server IP, but the install guide doesn't mention configuring a zone file.

**Comment by pboehm** [comment]
Hello,

thanks for using my little side project. I can describe the setup I use for `pboehm.de`, which should be applicable for your case. `pboehm.de` is registered at DomainFactory which supports nameserver delegation for subdomains. I have created an `A`-record `dns.pboehm.de` pointing to the VM ddns is running on. Then I have created an `NS`-record for `d.pboehm.de` pointing to `dns.pboehm.de`. This delegates the subdomain `d.pboehm.de` to my ddns instance which handles this subdomain and answers DNS-queries for hosts in this domain.

The backend of ddns is then executed with the following parameters:

```
ddns --soa_fqdn=dns.pboehm.de --domain=d.pboehm.de backend  
```

I hope this helps you setting up ddns.
